src/main/resources/sql/V10__Index_on_index_table.sql

```diff
@@ -0,0 +1,7 @@
+create index if not exists index_name on index (name);
+create index if not exists index_key on index (key);
+create index if not exists index_sha256hex on index (sha256hex);
+create index if not exists index_start_entry_number on index (start_entry_number);
+create index if not exists index_end_entry_number on index (end_entry_number);
+create index if not exists index_start_index_entry_number on index (start_index_entry_number);
+create index if not exists index_end_index_entry_number on index (end_index_entry_number);
```

Comment:
There are a lot of indexes here and all on individual columns. Do these indexes help with loading and/or querying? Do we still need them if we also have batching?